packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -388,6 +390,7 @@ const selectInputProps = computed(() => {
 
 .onyx-select-wrapper {
   @include layers.component() {
+    --max-height: 20rem;
```

Comment:
the var would probably be overridden directly on <OnyxSelect in the project right?
then maybe enhance the name so it's clear that the max height refers to the flyout part?